packages/icons/web/v2/share_16.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <g clip-path="url(#share_16_svg__a)">
```

Comment:
Ids in SVGs can lead to problems when multiple instances of the same icon are used on a single page. That's why we try to avoid them as much as possible. In this case, the clip-path isn't necessary.